tfidf-benchmark/Visualize.ipynb

```diff
@@ -0,0 +1,1558 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    groupy_sklearn
</pre><p>Remove unmaed:0 column (index in all probability) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>